### ISSUE TITLE
Show type info of yield

### ISSUE THREAD

**Issue by falsandtru** [microsoft/TypeScript#32283]
@rbuckton `yield` keyword should show its argument type and return type when a cursor is put on there.

**Comment by sandersn** [comment]
Good idea, although I don't think we produce yield's type when checking, although this might have recently changed.

**Comment by falsandtru** [comment]
In fact, finding a break of type checking such as #32248 (yield wrongly returns any type.) is impossible when we can't directly check a type of yield. When yield argument type is any, we can't know it. When yield return type is any, we can't know it without seeing an inferred type via a variable.

**Comment by falsandtru** [comment]
@sandersn @rbuckton I recommend you to release this feature with the next release to make complete user experience of enhanced generators.

**Comment by Kingwl** [comment]
What is the expected display? 
Similar to NewKeyword? or something like `(TNext) yield (T)`

**Comment by falsandtru** [comment]
I expect it.

**Comment by Kingwl** [comment]
Expect which one🤦🏻‍♂️

**Comment by falsandtru** [comment]
Ah, I meant `(TNext) yield (T)`.

**Comment by Kingwl** [comment]
@DanielRosenwasser Any suggestions？🙋🏻‍♂️

**Comment by DanielRosenwasser** [comment]
Similar to #11261 which is about quick info on `return`.
